x-pack/plugin/rank-rrf/src/yamlRestTest/resources/rest-api-spec/test/rrf/300_rrf_retriever.yml

```diff
@@ -0,0 +1,261 @@
+setup:
+  - skip:
+      features: close_to
```

Comment:
doesn't look like `close_to` feature is used  in this test.

Reply:
Removed. Good catch!